products/rhel8/CMakeLists.txt

```diff
@@ -9,15 +9,29 @@ set(DISA_SRG_TYPE "os")
 ssg_build_product(${PRODUCT})
 
 ssg_build_html_table_by_ref(${PRODUCT} "nist")
+set(NIST_REF_FORMAT "'.*-(\\\\d+)(?:\\\\((\\\\d+)\\\\))?'")
+ssg_build_html_ref_table("new-table-${PRODUCT}-nist" "${PRODUCT}" "nist" "${NIST_REF_FORMAT}")
 ssg_build_html_table_by_ref(${PRODUCT} "cui")
+set(CUI_REF_FORMAT "'(\\\\d+)(?:\\\\.(\\\\w+)(?:\\\\.(\\\\w+)(?:\\\\.(\\\\w+))?)?)?'")
+ssg_build_html_ref_table("new-table-${PRODUCT}-cui" "${PRODUCT}" "cui" "${CUI_REF_FORMAT}")
 ssg_build_html_table_by_ref(${PRODUCT} "cis")
+set(CIS_REF_FORMAT "'(\\\\d+)\\\\.(\\\\d+)(?:\\\\.(\\\\w+)(?:\\\\.(\\\\w+)(?:\\\\.(\\\\w+))?)?)?'")
```

Comment:
I wonder what the regexes are good for. Are there any references that don't match the regex? In this specific example, I believe you want to use all of the `cis@rhel8` references. Maybe you don't need them at all?

Reply:
The trick is that if I describe a reference using a regex with groups, those groups can be used when sorting them in a smart way.
But it looks terrible in the cmake config, and it also doesn't make sense to be repeated in all products.

Reply:
If I rephrase that, the purpose of these regular expressions is correct sorting of the items in the generated tables. Is that correct? And is there any other usage of these regular expressions?

Reply:
It has also a priority purpose - if there are more reference strings in that reference category, ones that match the regex are listed first. One can observe this on ANSSI BP/NT references.